src/protocols/bitcoin/libs/BitcoinAdapter.ts

```diff
@@ -1,11 +1,58 @@
 /* eslint-disable class-methods-use-this */
 
+import * as ecc from '@bitcoin-js/tiny-secp256k1-asmjs';
+import { BIP32Factory } from 'bip32';
+import * as bitcoin from 'bitcoinjs-lib';
```

Comment:
I would import specific parts to allow tree-shaking if it supported by bitcoinjs-lib